spec/models/review_spec.rb

```diff
@@ -0,0 +1,56 @@
+RSpec.describe Review, type: :model do 
+  context 'validations' do
+    %i[body score].each do |attr|
+      it { is_expected.to validate_presence_of(attr) }
+    end
+
+    it { is_expected.to allow_value("!#$%&'*+/=?^_`{|}~Qw., 1").for(:body) }
```

Comment:
usually it's better to add one more test with `is_expected.not_to allow_value('not allowed value')